docs/qa/check-lists/USER_AUTH_BE.md

```diff
@@ -0,0 +1,94 @@
+<ol>
+  <li>
+    Auth endpoints
+    <ol>
+      <li>
+        Register user: POST `/api/auth/register`
+        <ul>
+          <li>Body for request must be provided in the following format:</li>
+        </ul>
+        <pre>
+          <code>
+          {
+            "firstName": "David",
+            "lastName": "Kalachyan",
+            "username": "david",
+            "password": "david",
+            "address": {
+              "line": "line",
+              "city": "city",
+              "country": "country"
+            },
+            "email": "email@example.com"
+          }
+          </code>
+        </pre>
+        <ul>
+          <li>All parameters are mandatory - If one or more parameters are blank, the corresponding message in the response must be returned with a 400 code response.</li>
+          <li>Email parameter must be provided in the following format: "some@some.com" - If this parameter is in the wrong format, the corresponding message in the response must be returned with a 400 code response.</li>
+          <li>Authorization not needed for this endpoint - It is possible to perform the request with no authorization.</li>
+          <li>If all parameters are provided in the correct format, the response must return a JWT token for authorizing this user.</li>
+          <li>Users can authorize all next requests without performing additional AUTH requests (e.g., GET user by Id).</li>
+          <li>Users can authorize with registered username and password via the AUTH endpoint and obtain a JWT token.</li>
+        </ul>
+      </li>
+      <li>
+        Auth user: POST `/api/auth/authenticate`
+        <ul>
+          <li>It is possible to get a JWT token by providing the username and password in the body in the following format:</li>
+        </ul>
+        <pre>
+          <code>
+          {
+            "username": "{{username}}",
+            "password": "{{password}}"
+          }
+          </code>
+        </pre>
+        <ul>
+          <li>All these parameters are mandatory - If one or more parameters are blank, a 403 response code must be returned with no data (user enumeration isn't possible).</li>
```

Comment:
Why code 403?
This is request "bad" => 400

Or 200 with description "Wrong username or password"

Reply:
Fixed